### ISSUE TITLE
[BUG] Some times when a condition is removed the token stays invisible

### ISSUE THREAD

**Issue by Zamrod** [p4535992/conditional-visibility#4]
**Module Version:** v0.3.1

**Describe the bug**
When creating conditions using Combat Utility Belt for In Darkness or Invisible sometimes when the condition is removed the token doesn’t become visible again and it becomes stuck like that with no way to make it visible otherwise than disabling the module. 

This also is sometimes accompanied by an error:

```
helpers.mjs:586 
        
       Uncaught TypeError: Cannot set property isVisible of #<Token> which has only a getter
[Detected 1 package: conditional-visibility]
    at setProperty (helpers.mjs:586)
    at ConditionalVisibility.js:128
```

**Comment by p4535992** [comment]
patch the error on 0.3.2

**Comment by p4535992** [comment]
This issue is closed, because the new series 0.4.X is just to different, checkout the last version of the module.
